solutions/security/detect-and-alert/mitre-attandckr-coverage.md

```diff
@@ -32,6 +32,16 @@ You can map custom rules to tactics in **Advanced settings** when creating or ed
 :screenshot:
 :::
 
+Refer to the following table to find the MITRE ATT&CK® version that's mapped to your version of {{elastic-sec}}. 
+
+| MITRE ATT\&CK® version | {{elastic-sec}} version |
+| :---- | :---- |
+| [**v16.1**](https://attack.mitre.org/resources/updates/updates-october-2024/) | **9.0.0, 9.1.0** |
```

Comment:
@dplumlee I believe we'll need to make this if you're backporting https://github.com/elastic/kibana/pull/231375 to 9.1 and 9.0.

```suggestion
| [**v17.1**](https://attack.mitre.org/resources/updates/updates-october-2024/) | **9.0.0, 9.1.0** |
```

cc: @banderror @approksiu

Reply:
```suggestion
| [**v16.1**](https://attack.mitre.org/resources/updates/updates-october-2024/) | **9.0.0-9.0.6, 9.1.0-9.1.3** |
```